src/test/java/com/github/commonsrdf/dummyimpl/BlankNodeImpl.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.commonsrdf.dummyimpl;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+import com.github.commonsrdf.api.BlankNode;
+
+public class BlankNodeImpl implements BlankNode {
```

Comment:
Could you change all of the Impl classes that won't be called by external users to be package private to avoid exposing them in the public API? We are not relying on OSGi or Java 9 Modules/etc. where this wouldn't be an issue, but we can avoid exposing them where possible by not making them public.